client/src/containers/CampaignAdmin/tests/CampaignAdmin-test.js

```diff
@@ -1,25 +1,27 @@
 /* global jest, test, describe, beforeEach, it, expect */
 
-import { Component as CampaignAdmin } from '../CampaignAdmin';
 import { fireEvent, render, screen } from '@testing-library/react';
 import React from 'react';
-
+import { Component as CampaignAdmin } from '../CampaignAdmin';
 
 let mockResponse;
 
 function makeProps(obj = {}) {
   return {
     sectionConfig: {
-      reactRoutePath: '',
+      reactRoutePath: '/campaigns',
```

Comment:
Seems wrong that we're changing this in a linting PR, this should be reverted. The tests were passing before so it's not a good sign if we need to change this now.

Reply:
See comment https://github.com/silverstripe/silverstripe-campaign-admin/pull/271#discussion_r1253655654